rfqmsg/request.go

```diff
@@ -22,9 +22,11 @@ const (
 )
 
 type (
-	// requestSuggestedTickRate is a type alias for a record that represents
-	// the suggested rate tick for the quote request.
-	requestSuggestedTickRate = tlv.OptionalRecordT[tlv.TlvType4, uint64]
+	// requestSuggestedAssetRate is a type alias for a record that
+	// represents the suggested asset to BTC rate for the quote request.
+	//
+	// nolint: lll
```

Comment:
nit: could use the wrapped formatting as with `requestInAssetGroupKey` below instead.